tasks/configure.yml

```diff
@@ -14,24 +14,32 @@
         state: present
         update_cache: yes
 
+    - name: Set proper version for apt repo
+      ansible.builtin.set_fact:
+        k8s_cluster_apt_kubernetes_version: "{{ k8s_cluster_kubernetes_version | regex_replace('^(v\\d\\.\\d\\d)\\.\\d$', 'v\\1') }}"
```

Comment:
Maybe add comment illustrating how this regex works?

Reply:
Done